FRW/Resource/en.lproj/Localizable.strings

```diff
@@ -983,6 +983,19 @@
 "my_accounts" = "My Accounts";
 "Inaccessible" = "Inaccessible";
 "View Recovery Phrase" = "View Recovery Phrase";
+"import_google_desc" = "Retrieve the legacy backup from Google Drive.";
+"import_icloud_desc" = "Retrieve the legacy backup from iCloud.";
+"import_phrase_desc" = "Support 12 words seed phrase.";
+"import_keystore_desc" = "Import keystore file (Blocto Backup).";
+"import_privatekey_desc" = "Import with raw hex key.";
+"keystore_json" = "You can import the json file from other wallet (eg. Blockto)";
+"keystore_password" = "Enter password for json file";
+"keystore_address" = "Enter your flow address (Optional)";
+"x_account_found_title" = "%@ Accounts Found";
+"no_account_found" = "No Account Found";
+"import_x_wallet" = "Import %@ Wallet";
```

Comment:
Can you give a demo？ @jeden

Reply:
After looking at the code, I realized that this resource is used only with the "1" value, so instead of pluralize it, it could be simplified by just changing it to:
```suggestion
"import_x_wallet" = "Import 1 Wallet";
```

However pluralization can be applied to `"no_account_found"` and `"x_account_found_title"`. I've pushed to the `feat/private-key-import+stringsdict` branch an implementation that uses the stringsdict; we can talk later if needed.